### PR TITLE
tinc playbook

### PR DESCRIPTION
**Pull request by jschaul**

### CODE REVIEW

File: ansible/README.md [hunk 1]
```diff
@@ -175,6 +175,25 @@ poetry run ansible-playbook -i hosts.ini restund.yml -vv
 
 ### tinc
 
-* (optional) add a `vpn_ip=Z.Z.Z.Z` item to each entry in the hosts file with a (fresh) IP range if you wish to use [tinc mesh vpn](http://tinc-vpn.org/). Ensure to run the tinc.yml playbook first. See the Tinc section for details.
+Installing [tinc mesh vpn](http://tinc-vpn.org/) is **optional and experimental**. It allows having a private network interface `vpn0` on the target VMs.
 
-TODO add playbook.
+* Add a `vpn_ip=Z.Z.Z.Z` item to each entry in the hosts file with a (fresh) IP range if you wish to use  Ensure to run the tinc.yml playbook first, before other playbooks.
```
Comment (fisx) [inline ansible/README.md]: first sentence is missing something

File: ansible/README.md [hunk 2]
```diff
@@ -175,6 +175,25 @@ poetry run ansible-playbook -i hosts.ini restund.yml -vv
...
+```
+[all]
+server1 ansible_host=X.X.X.X vpn_ip=10.10.1.XXX
+
+[cassandra]
+server1
+
+[vpn:children]
+cassandra
+elasticsearch
```
Comment (fisx) [inline ansible/README.md]: why just these two?  as an example?

File: ansible/README.md (hunk 1 above)
Comment (jschaul) [reply ansible/README.md]: thanks, fixed.

File: ansible/README.md (hunk 2 above)
Comment (jschaul) [reply ansible/README.md]: as an example, yes - rewrote this part slightly to make it (hopefully) a little clearer.

**Pull request merged**